cilantro_audit/view_audit_templates.py

```diff
@@ -0,0 +1,50 @@
+import kivy
+from kivy.app import App
+from kivy.clock import Clock
+from kivy.uix.button import Button
+from kivy.uix.screenmanager import Screen
+from kivy.uix.screenmanager import ScreenManager
+from mongoengine import connect
+
+from cilantro_audit.audit_template import AuditTemplate
+# Required version
+kivy.require(KIVY_REQUIRED_VERSION)
```

Comment:
After pulling the repo, references to constants like `KIVY_REQUIRED_VERESION` do not exist. Is this branch updated with master?

Reply:
It isn't, I just changed like 4 lines of code to make Bader's work with the database. It must not have been merged with master's updated code in a bit.

Reply:
Yeah. That's causing some issues now. It seems like I can't run the code any longer. I guess this give us two options: merge the current branch with master regardless, or merge master with the current branch then, merge it back to master after checking it's all clear and good to go.